README.md

```diff
@@ -1,2 +1,43 @@
-# Website
-This repo is work in progress for the official website of **[Kubesimplify](https://kubesimplify.github.io/website/)**
+<h1 align="center"><a href="https://kubesimplify.github.io/website/">Kubesimplify Website </a></h1>
+<p align="center"><i>The source code for Kubesimplify.com</i></p>
+
+This repository contains the assets required to build the Kubernetes website and documentation. We're glad that you want to contribute! 
+
+- [Code of Conduct ](code-of-conduct.md)
+
+<h1>Contributing Guidelines: </h1>
+
+Kubesimplfy's new website. This codebase is what runs on [Kubesimplify](https://kubesimplify.github.io/website/). 
+
+
+For new developers getting started, run the following in your terminal:
+
+Fork the repo by clicking the fork button. This will clone the repo in your account, changes done here will be shown in the main repo. 
+ 
+Clone the repo to your local device: 
+```bash
+   git clone https://kubesimplify.github.io/website/ 
+   cd WEBSITE-1
+```
+Install dependencies:
+```bash
+    $ npm install docusaurus
```

Comment:
@pavangudiwada @verma-kunal do we need to explicitly mention docusaurus or normal `npm install` works?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
to run the code on local host we will be needing docusaurus installed

Reply:
Just `npm install` will automatically install all the dependencies

Reply:
yeah I feel the same

Reply:
the site will not run as we are using docusaurus you can check it out

Reply:
It's running

Reply:
Before: <img width="1440" alt="Screenshot 2022-05-08 at 10 10 37 PM" src="https://user-images.githubusercontent.com/90847875/167306279-7dd351fd-131e-48da-84dc-27da411fd96e.png">
As you can see after deleting the docusaurus modules it stucks in the terminal and doesnt redirects directly to localhost as we want.
After Docusaurus is installed: 
<img width="1440" alt="Screenshot 2022-05-08 at 10 15 52 PM" src="https://user-images.githubusercontent.com/90847875/167306425-10f28c4d-79fa-406a-bb85-5c39c98dc664.png">

Reply:
The "before" screenshot in your comment shows the `npm run` command without specifiying any script. Hence, it listed the available scripts for `npm run` (refer [npm-run-script documentation](https://docs.npmjs.com/cli/v8/commands/npm-run-script))

The step in question here is `npm install` which is expected to read our [package-lock](https://github.com/kubesimplify/website/blob/main/package-lock.json) file and install all the dependencies including docusaurus as it's mentioned in the package-lock file. (refer [npm install documentation](https://docs.npmjs.com/cli/v6/commands/npm-install))

Here's a block copy pasted from my shell showing `npm install` is working as expected and compilation was successfull

```BASH
[anubhav@anubhav website-main]$ ls
babel.config.js  code-of-conduct.md  docs  docusaurus.config.js  LICENSE  package.json  package-lock.json  README.md  sidebars.js  src  static
[anubhav@anubhav website-main]$ npm install

added 1164 packages, and audited 1165 packages in 8s

193 packages are looking for funding
  run `npm fund` for details

21 vulnerabilities (4 moderate, 17 high)

To address issues that do not require attention, run:
  npm audit fix

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
[anubhav@anubhav website-main]$ npm start

> website@0.0.0 start
> docusaurus start


   ╭──────────────────────────────────────────────────────────────────────────────────────────╮
   │                                                                                          │
   │                      Update available 2.0.0-beta.18 → 2.0.0-beta.20                      │
   │                                                                                          │
   │     To upgrade Docusaurus packages with the latest version, run the following command:   │
   │             `npm i @docusaurus/core@latest @docusaurus/preset-classic@latest`            │
   │                                                                                          │
   ╰──────────────────────────────────────────────────────────────────────────────────────────╯

[INFO] Starting the development server...
Warn: `blogDir` doesn't exist: "/home/anubhav/Downloads/website-main/blog".
[SUCCESS] Docusaurus website is running at http://localhost:3000/website/.

✔ Client
  Compiled successfully in 5.97s

client (webpack 5.72.0) compiled successfully


```